src/machine/machinectl.c

```diff
@@ -899,6 +899,100 @@ static int show_machine(int argc, char *argv[], void *userdata) {
         return r;
 }
 
+static int print_image_hostname(sd_bus *bus, const char *name) {
+        _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+        const char *hn;
+        int r;
+
+        r = sd_bus_call_method(
+                        bus,
+                        "org.freedesktop.machine1",
+                        "/org/freedesktop/machine1",
+                        "org.freedesktop.machine1.Manager",
+                        "GetImageHostname",
+                        NULL, &reply, "s", name);
+        if (r < 0)
+                return r;
+
+        r = sd_bus_message_read(reply, "s", &hn);
+        if (r < 0)
+                return r;
+
+        if (!isempty(hn))
+                printf("\tHostname: %s\n", hn);
```

Comment:
the rest of the code works like that too: use as many \t as fit in, then fill up with spaces, so things are properly aligned. Or to say this differently: in this output the `:` is currently aligned to the 16th character of each line. Since "Hostname" has 8 chars, if we prefix it with one TAB we are at 16 chars, as desired. "Machine ID" otoh has 10 chars, hence we can't prefix with TABs anymore, and instead use 6 spaces, to reach the 16 chars...

Of course, it's valid to question whether to use tabs at all here, but If we change that we should probably do that in a separate PR that drops `\t` all across the board, and not just here...

Reply:
Ack.